### PR TITLE
Added new RL dashboard content

### PR DESCRIPTION
**Pull request by christinechoy-okta**
## Description:
- **What's changed?** Added new content to RL dashboard
- **Is this PR related to a Monolith release?** Yes, Release 2021.12.0

### Resolves:

* [OKTA-450191](https://oktainc.atlassian.net/browse/OKTA-450191)

### CODE REVIEW

File: packages/@okta/vuepress-site/docs/reference/rl-dashboard/index.md [hunk 1]
```diff
@@ -8,53 +8,97 @@ excerpt: >-
...
+
+#### Violation graphs
+
+The Violations line graph plots the number of violations over the selected time period, which can be the last 24 hours, last 7 days, last month, or last 3 months. The Violations per endpoint bar graph shows the top ten number of violations by endpoint over the same time period.
+
+You can hover over the line graph or bar graph to get more details. The details also include a link to the system log where you can view filtered events specific to that endpoint and time period.
+
+#### APIs table
+
+The APIs table shows each endpoint and its corresponding rate limit for your org. The rate limits include all overrides, dynamic scale, or Workforce multipliers. The table also shows the number of violations and warnings for each endpoint in the last hour, and includes a trendline of usage in the past hour.
```
Comment (brentschaus-okta) [inline packages/@okta/vuepress-site/docs/reference/rl-dashboard/index.md:49]: Is there a grammatical reason for using both "last hour" and "past hour"? My mind anticipated a parallel structure. Like this:
```suggestion
The APIs table shows each endpoint and its corresponding rate limit for your org. The rate limits include all overrides, dynamic scale, or Workforce multipliers. The table also shows the number of violations and warnings for each endpoint in the last hour, and includes a trendline of usage in the last hour.
```

File: packages/@okta/vuepress-site/docs/reference/rl-dashboard/index.md [hunk 2]
```diff
@@ -8,53 +8,97 @@ excerpt: >-
...
+
+You can sort on all the columns. The Trendline is sorted by the current usage.
+
+Each endpoint is a link that navigates you to the [dashboard view of endpoint rate limits and violations](#endpoint-rate-limits-and-violations) where you can obtain more details on the endpoint.
+
+You can perform a search or filter your table list by making an entry in the Search textbox. You can also perform a "lookup" with the Search textbox, for example, in cases when you're calling Okta with an example `requestURI` and you want to know the rate limit that will be applied to calls that are made against that `requestURI`.
```
Comment (brentschaus-okta) [inline packages/@okta/vuepress-site/docs/reference/rl-dashboard/index.md:55]: The 2nd sentence here is rather long. My mind naturally edited into two shorter sentences lol

File: packages/@okta/vuepress-site/docs/reference/rl-dashboard/index.md [hunk 3]
```diff
@@ -8,53 +8,97 @@ excerpt: >-
...
+
+### Change rate limit settings
+
+#### Client-based rate limiting
+
+You can enable per-client rate limiting to prevent a single client from blocking Internet traffic to Okta's `/login/login.htm` and OAuth 2.0 `/authorize` endpoints. For each of these endpoints, select from these options:
+
+* **Enforce and log per client (recommended)** - Enables client-based rate limiting for all clients
```
Comment (brentschaus-okta) [inline packages/@okta/vuepress-site/docs/reference/rl-dashboard/index.md:63]: I still haven't sussed out whether we use colons or hyphens after items in a list. See this list from the previous page in the Rate Limits section: https://developer.okta.com/docs/reference/rate-limits/#other-applicable-rate-limit-content

**Review by brentschaus-okta — APPROVED** [review]
LGTM!

Just a few comments and queries.

File: packages/@okta/vuepress-site/docs/reference/rl-dashboard/index.md (hunk 1 above)
Comment (christinechoy-okta) [reply packages/@okta/vuepress-site/docs/reference/rl-dashboard/index.md:49]: 👍

File: packages/@okta/vuepress-site/docs/reference/rl-dashboard/index.md (hunk 2 above)
Comment (christinechoy-okta) [reply packages/@okta/vuepress-site/docs/reference/rl-dashboard/index.md:55]: Fixed.

File: packages/@okta/vuepress-site/docs/reference/rl-dashboard/index.md (hunk 3 above)
Comment (christinechoy-okta) [reply packages/@okta/vuepress-site/docs/reference/rl-dashboard/index.md:63]: I used the em-dash (`&mdash;` ).
Thanks for pointing that out.

**Comment by joec-okta** [comment]
Looks good to me.

**Review by joec-okta — APPROVED** [review]
LGTM

**Pull request merged**